### DISCUSSION TITLE
大佬，你瑟图API挂了

### DISCUSSION THREAD

**Discussion by swh20021102** [Tsuk1ko/CQ-picfinder-robot#306] [问题求助]
Application error
An error occurred in the application and your page could not be served. If you are the application owner, [check your logs for details](https://devcenter.heroku.com/articles/logging#view-logs). You can do this from the Heroku CLI with the command
heroku logs --tail

**Comment by Tsuk1ko** [comment]
有憨批CC

![image](https://user-images.githubusercontent.com/24877906/162606772-79fbddad-c66c-429f-bb63-bcd56081451a.png)
